allennlp/service/predictors/semantic_role_labeler.py

```diff
@@ -48,6 +48,10 @@ def _json_to_instance(self, json: JsonDict) -> Instance:
         raise RuntimeError("this should never be called")
 
     @overrides
+    def _batch_json_to_instance(self, json: List[JsonDict]) -> List[Instance]:
+        raise NotImplementedError("The SRL Predictor does not currently support batch prediction.")
```

Comment:
This seems like a pretty good argument for just having this use `_json_to_instance` - what's the reason we can't do that?

Reply:
We want to have the sentence and verbs in the output, but we don't want to have to extract them from the model.